### DISCUSSION TITLE
Given that we have shlinkio server running local, can access on localhost:8080 with this docker-compose.yml, can we use the web client?

### DISCUSSION THREAD

**Discussion by johnsmithodom** [shlinkio/discussions#26] [Help wanted] [answered]
#### Shlink web client version

*Version: v3.10.1
* How do you use shlink-web-client: app.shlink.io|Docker image|self-hosted
Docker image and self-hosted
#### Summary
Current shlinkio is accessable on localhost:8080, with such config, can we get the api key for the web client?
 

Not sure go the get the api key to set up the web client.

**Comment by acelaya** [comment, marked as answer]
Please, try to search in the documentation before posting basic questions here.

The first result when searching "API key" is this https://shlink.io/documentation/api-docs/authentication/#managing-api-keys
